pyproject.toml

```
@@ -125,6 +125,7 @@ known_golem = "goth, yapapi, ya_activity, ya_market, ya_net, ya_payment"
 sections = ["FUTURE", "STDLIB", "THIRDPARTY", "GOLEM", "FIRSTPARTY", "LOCALFOLDER"]
```

```suggestion
sections = ["FUTURE", "STDLIB", "THIRDPARTY", "GOLEM", "FIRSTPARTY", "LOCALFOLDER"]
line_length = 100
```
As isort profile uses fixed line length on black profile, we need to override that too.